.prettierrc.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-    trailingComma: 'all',
+    trailingComma: 'es5',
```

Comment:
Bringing the base prettierrc in-line with static

Reply:
I personally think trailing commas make for easier to read diffs (as you don't have a diff on the line with the new, added comma)